AP-3432: Update ProceedingType list with Public Law Family codes

colinbruce opened the pull request:
<!--Ticket-->

https://dsdmoj.atlassian.net/browse/AP-3432

This PR adds
* new, Public Law Family, proceeding types to the CFEConstants
* updates the MockThresholdWaivers with the new prefix
* regenerated swagger docs, including the new proceeding_code values

This will allow Civil Apply to send CFE data containing PLF proceeding types.  At this time we do not expect PLF to require and calculation or waiver changes

---

## Checklists

Author: (before you ask people to review this PR)

- [x] Diff - review it, ensuring it contains only expected changes
- [x] Whitespace changes - avoid if possible, because they make diffs harder to read and conflicts more likely
- [ ] Changelog - add a line, if it meets the criteria
- [x] Secrets - no secrets should be added
- [x] Commit messages - say *why* the change was made
- [x] PR description - summarizes *what* changed and *why*, with a JIRA ticket ID
- [x] Tests pass - on CircleCI
- [x] Conflicts - resolve if Github reports them. e.g. with `git rebase main`

Reviewers remember:

- [ ] Jira ticket criteria are met
- [ ] Migrations - test migration and rollback: `rake db:migrate && rake db:rollback`

willc-work commented on app/constants/cfe_constants.rb:
```diff
@@ -4,20 +4,25 @@ module CFEConstants
   VALID_ASSESSMENT_VERSIONS = %w[6 7].freeze
 
-  # Valid CCMS Codes for proceeding types - probably need to get this from LFA in future
+  # Valid CCMS Codes for proceeding types - probably need to gPUBLIC_LAW_FAMILY_CCMS_CODESet this from LFA in future
```
is this a copy and paste error?

colinbruce replied:
It looks like it doesn't it! 😆 
I'll resolve that too :+1:

colinbruce commented:
I have rebased on main (to incorporate the intervening work, and added an update to remove 4 non-means-tested proceedings from the new matter type.  Civil Apply should never send them, so CFE should raise an error if it is ever sent them
